src/components/data/listing/Listing.js

```diff
@@ -92,7 +85,18 @@ function Listing(props) {
 
     useEffect(() => {
         setSelected([]);
-    }, [path]);
+        if (path) {
```

Comment:
when landing on `/data` or `/data/ds`, the `path` is null, which resulted in the error.

Reply:
I also found another bug where when you logged out, and land on `/data `or `/data/ds`, the path sent to data-info get appended with key-cloak query params. I haven't fixed this issue yet.